Products/CMFPlone/browser/login/templates/insufficient_privileges.pt

```diff
@@ -1,32 +1,42 @@
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
-      xmlns:tal="http://xml.zope.org/namespaces/tal"
-      xmlns:metal="http://xml.zope.org/namespaces/metal"
-      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
-      lang="en"
-      metal:use-macro="context/@@main_template/macros/master"
-      i18n:domain="plone">
+<html
+    xmlns="http://www.w3.org/1999/xhtml"
+    xml:lang="en"
+    xmlns:tal="http://xml.zope.org/namespaces/tal"
+    xmlns:metal="http://xml.zope.org/namespaces/metal"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    lang="en"
+    metal:use-macro="context/@@main_template/macros/master"
+    i18n:domain="plone">
 
-<head>
-    <metal:block fill-slot="top_slot"
-                 tal:define="dummy python:request.set('disable_border',1);
-                             disable_column_one python:request.set('disable_plone.leftcolumn',1);
-                             disable_column_two python:request.set('disable_plone.rightcolumn',1);" />
-</head>
+  <head>
+    <metal:block
+        fill-slot="top_slot"
+        tal:define="dummy python:request.set('disable_border',1);
+                    disable_column_one python:request.set('disable_plone.leftcolumn',1);
+                    disable_column_two python:request.set('disable_plone.rightcolumn',1);" />
+  </head>
 
-<body>
-   <metal:content fill-slot="main">
-      <h1 class="documentFirstHeading"
+  <body>
+    <metal:content fill-slot="main">
+      <h1
+          class="documentFirstHeading"
```

Comment:
I am no fan of putting the class on the next line when it fits perfectly on the original.
Is this some automatic change by your editor, or a command line tool that you called?

It's okay, but it distracts from the real fix, which is just one line. Well, now I see you at least put that in a separate commit, which helps.

Reply:
@mauritsvanrees thats come from my vscode settings, i use the redhat xml formatter extension to format html/tal/xml templates. Any other settings preferred?

Reply:
Lately I see more packages using [`zpretty`](https://pypi.org/project/zpretty/), created by @ale-rt. When I try this tool on this template on your branch or master, there are again lots of changes in both cases. It seems a bit closer to master though.

I would suggest to remove the first commit from this PR, to avoid changing lots of lines now, and then again in a different way layer.
But your fix is more important than squabbling over formatting, so feel free to ignore this suggestion. :-)

There is one failure on Jenkins 3.7, but that looks unrelated, so should be fine to merge.